test/fixtures/github/workflow-call.yaml

```diff
@@ -0,0 +1,14 @@
+name: workflow-call
+
+jobs:
+  call-local-workflow:
+    uses: ./../fixtures/github/dependant-jobs.yaml
```

Comment:
can you check workflow call on remote?

Reply:
I am not sure I understand the question - in line 10 of this file we have a check for remote workflow call